R/iso_osrm.R

```diff
@@ -0,0 +1,61 @@
+#' Calculate voronoi-style polygons based on grid with OSRM routing duration
+#'
+#' @note The function requires OSRM to be running, with the server being
+#' @note defined by the setting for the osrm package,
+#' @note i.e., getOption("osrm.server") and getOption("osrm.profile")
+#'
+#'
+#' @param x An sf object of a grid
+#' @param points An sf object of points
+#' @param measure passed on to osrm::osrmTable, can be either "duration" (minutes) or "distance" (meters)
+#' @param osrm.server the base URL of the routing server (end with "/"!)
+#' @param osrm.profile the routing profile to use
+#'
+#' @return An sf object of a grid, where column 'index_min' gives the index of the input point, for which the measure is minimal
+#' @importFrom  rlang .data
+#' @export
+#'
+#' @examples
+#' points = points_oldenburg
+#' x = sf::st_as_sf(iso_grid(net_oldenburg_raw, resolution = 10))
+#' grid = isonoi::iso_osrm(x, points, measure = "duration",
+#' osrm.server = "http://router.project-osrm.org/", osrm.profile = "car")
+#' plot(grid)
+iso_osrm <- function(x,
+                     points,
+                     measure = "duration",
+                     osrm.server = getOption("osrm.server"),
```

Comment:
Where is this first defined?

Reply:
You mean the `osrm.server` option? That option is defined as part of the **osrm** package. How should this be referenced instead?